packages/sst/src/constructs/NextjsSite.ts

```diff
@@ -423,18 +423,25 @@ export class NextjsSite extends SsrSite {
   }
 
   private wrapHandler() {
-    const { path: sitePath } = this.props;
+    const { path: sitePath, experimental } = this.props;
     const wrapperName = "nextjssite-index";
     const serverPath = path.join(sitePath, ".open-next", "server-function");
 
     fs.writeFileSync(
       path.join(serverPath, `${wrapperName}.mjs`),
-      [
-        `import { handler as rawHandler } from "./index.mjs";`,
-        `export const handler = (event, context) => {`,
-        `  return rawHandler(event, context);`,
-        `};`,
-      ].join("\n")
+      experimental?.streaming
+        ? [
+            `import { handler as rawHandler } from "./index.mjs";`,
+            `export const handler = awslambda.streamifyResponse(async (...args) => {`,
```

Comment:
This will likely not work, or at least cause some issues.
In open-next when the streaming flag is there, we already add `awslambda.streamifyResponse` to the raw handler.

Reply:
hmm.. would this work?
```ts
export const handler = (...args) => {
  return rawHandler(...args);
};
```

Reply:
I was able to get this working using `awslambda.streamifyResponse`, it should work b/c it's just an additional callback... but let me confirm again

Reply:
Before you merge, let me verify the actual streaming response is behaving correctly, eg SSE

Reply:
Yeah it's ready to be tested. Do u need a snapshot release?

Reply:
Nope, I'm editing the node_modules directly

Reply:
Ok streaming works, LGTM